src/screens/TopicPageScreen/elements/TopicDomainHeader.js

```diff
@@ -88,13 +89,13 @@ const styles = StyleSheet.create({
     textAlign: 'left',
     color: isHeaderHide ? COLORS.white : COLORS.blackgrey
   }),
-  seeMemberText: {
+  seeMemberText: (followType) => ({
     fontSize: normalizeFontSize(12),
     fontFamily: fonts.inter[500],
     textAlign: 'left',
-    color: COLORS.blue,
+    color: followType === 'incognito' ? COLORS.anon_secondary : COLORS.blue,
     marginTop: normalize(1)
-  }
+  })
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `followType` prop is used to conditionally set the color of the text. This is a good use of conditional rendering based on props. However, it would be better to define the possible values for `followType` in PropTypes validation to ensure that only expected values are passed.
```diff
- followType: PropTypes.string
+ followType: PropTypes.oneOf(['incognito', 'public'])
```
This change will help catch any potential bugs where an unexpected value might be passed as `followType`.


<!-- This is an auto-generated comment by OSS CodeRabbit -->